lib/std/http/Server.zig

```diff
@@ -636,11 +700,10 @@ pub const HeaderStrategy = union(enum) {
 };
 
 /// Accept a new connection and allocate a Response for it.
-pub fn accept(server: *Server, options: HeaderStrategy) AcceptError!*Response {
+pub fn accept(server: *Server, options: HeaderStrategy) AcceptError!Response {
```

Comment:
You may remove the comment ` and allocate a Response for it.` because `accpet()` does not allocate.